Formula/i/iipsrv.rb

```diff
@@ -0,0 +1,39 @@
+class Iipsrv < Formula
+  desc "High performance image server for high resolution and scientific images"
+  homepage "https://iipimage.sourceforge.io"
+  url "https://downloads.sourceforge.net/iipimage/IIP%20Server/iipsrv-1.2/iipsrv-1.2.tar.bz2"
+  sha256 "d2483313d62eb617d05f3a55c5f09551fe5165725faba63aca680cb87d56bf6c"
+  license "GPL-3.0-or-later"
+
+  depends_on "pkg-config" => :build
+  depends_on "fcgi"
+  depends_on "jpeg-turbo"
+  depends_on "libmemcached"
+  depends_on "libpng"
+  depends_on "libtiff"
+  depends_on "openjpeg"
+  depends_on "webp"
+
+  def install
+    # Temporary fix for clang16
+    inreplace "configure", "TIFFGetVersion()", "int a"
```

Comment:
Please submit this upstream and indicate when it can be removed

Reply:
OK